### PR TITLE
Lazy documentation to clarify non-obvious code snippets.

### PR DESCRIPTION
**Pull request by emanueljoivo**

### CODE REVIEW

File: src/main/java/org/fogbowcloud/app/jes/arrebol/JobSynchronizer.java [hunk 1]
```diff
@@ -1,60 +1,73 @@
...
-            try {
-                String jobExecutionJson = this.requestsHelper.statusArrebolJob(executionId);
-                logger.debug("JSON Response [" + jobExecutionJson + "]");
-                Gson gson = new Gson();
-                ArrebolJob arrebolJob = gson.fromJson(jobExecutionJson, ArrebolJob.class);
-                this.updateJob(job, arrebolJob);
-            } catch (Exception e) {
-                throw new JobStatusException(e.getMessage());
+            JobExecArrebol jobExecArrebol = null;
+            while (Objects.isNull(jobExecArrebol)) {
```
Comment (wesleymonte) [inline src/main/java/org/fogbowcloud/app/jes/arrebol/JobSynchronizer.java]: Why needed this while?

File: src/main/java/org/fogbowcloud/app/utils/CustomFileAppender.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
+/** Utility class for Swagger configuration. */
```
Comment (wesleymonte) [inline src/main/java/org/fogbowcloud/app/utils/CustomFileAppender.java]: Not used for Swagger, but for log4j.

Comment (emanueljoivo) [reply src/main/java/org/fogbowcloud/app/utils/CustomFileAppender.java]: Solved in 41d40be35da627da99f10f2df4ac6fc8c95bcb09.

File: src/main/java/org/fogbowcloud/app/jes/arrebol/JobSynchronizer.java (hunk 1 above)
Comment (emanueljoivo) [reply src/main/java/org/fogbowcloud/app/jes/arrebol/JobSynchronizer.java]: To retry the status operation of a job and only stop trying when there is a response from Arrebol.

Comment (emanueljoivo) [reply src/main/java/org/fogbowcloud/app/jes/arrebol/JobSynchronizer.java]: Solved in 89f1f1d7ac916357cd8819e7f6200c1eb40fb875.

**Pull request merged**